src/Extension/ExtensionManager.php

```diff
@@ -283,7 +283,13 @@ public function getEnabledExtensions()
      */
     public function extend(Container $app)
     {
-        foreach ($this->getEnabledExtensions() as $extension) {
+        $extensions = new Collection($this->getEnabledExtensions());
```

Comment:
I would suggest that the sorted collection comes from `getEnabledExtensions()` instead of being used here.

Reply:
`getEnabledExtensions` is currently an array, it needs to be converted into a collection before we can sort it (unless you want an array `usort` instead)